src/tests/JIT/Regression/JitBlue/Runtime_113321/Runtime_113321.csproj

```diff
@@ -0,0 +1,8 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <Optimize>False</Optimize>
```

Comment:
Is this intended? Does this repro in the MinOpts?

Reply:
No it's not intended. Good catch.

Reply:
```suggestion
    <Optimize>True</Optimize>
```